.ruby-version

```diff
@@ -0,0 +1 @@
+2.6.6
```

Comment:
Is there a reason for this not being a 2.7 version?

Reply:
Fair. I've tweaked that now.